workflow-starfusion/workflow/config/workflow.ini

```diff
@@ -0,0 +1,19 @@
+output_dir=seqware-results
+output_prefix=./
+manual_output=false
+tmp_dir=tmp
+
+
+
+#Reference files
+ref_genome=/oicr/local/analysis/sw/starfusion/STAR-Fusion-v1.4.0/genomes/GRCh37_v19_CTAT_lib_Feb092018/ctat_genome_lib_build_dir
+
+#starfusion 
+starfusion_mem=64
+
+#programs
+PERL=/oicr/local/analysis/sw/perl/perl-5.22.2-tgl
+TABIX=/oicr/local/analysis/sw/tabix/tabix-0.2.6
+STAR=/oicr/local/analysis/sw/star/STAR-2.6.0c/bin/Linux_x86_64_static
+SAMTOOLS=/oicr/local/analysis/sw//samtools/samtools-0.1.17
```

Comment:
Do we need to use this old version of samtools? Please disregard if that's a requirement for this combination of tools

Reply:
I have tested the workflow with this version of samtools and it runs successfully.

Reply:
This version of samtools is at least 5 years old. Could you try with a more up to date version?

Reply:
I updated it to samtools 1.2